src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -3144,6 +3144,27 @@ End Class
             End Using
         End Function
 
+        <WorkItem(36546, "https://github.com/dotnet/roslyn/issues/36546")>
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestDoNotDismissIfEmptyOnBackspaceIfStartedWithBackspace(completionImplementation As CompletionImplementation) As Task
```

Comment:
💡 Can you also make sure there is a test for pressing <kbd>Ctrl</kbd>+<kbd>Space</kbd> followed by _multiple_ backspace characters to delete the entire word (e.g. press space twice to delete `Wr`)?